src/renderer/src/scenes/dir-select-scene/DirSelectScene.tsx

```diff
@@ -44,38 +61,75 @@ export default function DirSelectScene() {
 
   return (
     <div class="relative grid h-screen place-items-center p-8" style={{ background: GRADIENT }}>
+      <div
+        class="absolute top-0 left-0 w-full h-[50px] flex items-center justify-end"
+        style={{ "-webkit-app-region": "drag" }}
+      >
+        {os() !== "darwin" && <WindowControls />}
```

Comment:
We have a new window control component on the sidebar PR, can we use it instead?

https://github.com/Team-BTMC/osu-radio/pull/88/files#diff-8024e544834ad84e4465d5de0b24d7c8a13a2ec6bc11609e1d1be3ad387a0f38